csxtools/image_corr.py

```diff
@@ -7,37 +7,42 @@
 
 
 def correct_events(evs, data_key, dark_images, drop_raw=False):
-    out_data_key = data_key + '_corrected'
+    out_data_key = data_key + "_corrected"
     ev0 = next(evs)
-    new_desc = dict(ev0['descriptor'])
-    new_desc['data_keys'][out_data_key] = dict(new_desc['data_keys'][data_key])
-    new_desc['data_keys'][out_data_key]['source'] = 'subtract_background'
-    new_desc['uid'] = uuid.uuid4()
+    new_desc = dict(ev0["descriptor"])
+    new_desc["data_keys"][out_data_key] = dict(new_desc["data_keys"][data_key])
+    new_desc["data_keys"][out_data_key]["source"] = "subtract_background"
+    new_desc["uid"] = uuid.uuid4()
     if drop_raw:
-        new_desc['data_keys'].pop(data_key)
-    for ev in chain((ev0, ), evs):
-        new_ev = {'uid': str(uuid.uuid4()),
-                  'time': ttime.time(),
-                  'descriptor': new_desc,
-                  'seq_no': ev['seq_no'],
-                  'data': dict(ev['data']),
-                  'timestamps': dict(ev['timestamps'])}
-        corr, gain_img = subtract_background(ev['data'][data_key], dark_images) # noqa F821 TODO
-        new_ev['data'][out_data_key] = corr
-        new_ev['timestamps'][out_data_key] = ttime.time()
+        new_desc["data_keys"].pop(data_key)
+    for ev in chain((ev0,), evs):
+        new_ev = {
+            "uid": str(uuid.uuid4()),
+            "time": ttime.time(),
+            "descriptor": new_desc,
+            "seq_no": ev["seq_no"],
+            "data": dict(ev["data"]),
+            "timestamps": dict(ev["timestamps"]),
+        }
+        # TODO: replace stub with actual subtract_background implementation
```

Comment:
```suggestion
        # TODO (nisarnk): replace stub with actual subtract_background implementation
```
C++ Linters make you assign TODOs, and I like that habit :)

Reply:
done